frontend/src/metabase/ui/components/inputs/SegmentedControl/SegmentedControl.styled.tsx

```diff
@@ -21,9 +21,6 @@ export const getSegmentedControlOverrides =
             padding: fullWidth
               ? `${theme.spacing.sm} ${theme.spacing.md}`
               : theme.spacing.sm,
-            fontSize: theme.fontSizes.md,
-            fontWeight: "normal",
-            lineHeight: "1rem",
```

Comment:
the only reason we needed this is because we were using a `ReactNode` label in `PreviewModeSelector` and the line heights got messed up. I've fixed the issue within the `PreviewModeSelector` component itself - we shouldn't prescribe fixes like this to a global design system